src/constants/options/leaderboard.js

```diff
@@ -0,0 +1,3 @@
+export default ['day', 'week', 'month'];
+
+export const VALUE = ['Daily', 'Weekly', 'Montly'];
```

Comment:
can you move to language file

Reply:
Also make sure spelling is correct, “Monthly”, sublime has spell checker plugin if you use sublime 😉